frontend/device/input.lua

```diff
@@ -207,6 +207,12 @@ function Input:adjustTouchTranslate(ev, by)
         end
     end
 end
+function Input:adjustTouchAlyssum(ev)
+    ev.time = TimeVal:now()
+    if ev.type == EV_ABS and ev.code == ABS_MT_TRACKING_ID then
```

Comment:
`ev.time = TimeVal:now()` changes the time stamp in events so that  they have the same format as the `tv_now` in `Input:waitEvent` (which is also obtained by `TimeVal:now()`). We have discussed this. This will make the time of the events not so accurate, but I don't see a problem, and I don't know another way to solve it..

Then we have to change the finger id so that it starts from 0 (on Kobo Glo HD it starts from 1). Otherwise two finger gestures can't be dectected (in `gesturedetector.lua` it seems to check only `slot=0` and `slot=1`, if I understand correctly). This is new.

I couldn't figure out how to check if the device is Glo HD in `input.lua` so did things in `device.lua`. I had no knowledge in Lua, though could read it. Please feel free to change it to a better way.